static/js/ui_init.js

```diff
@@ -158,6 +159,7 @@ function initialize_bottom_whitespace() {
 function initialize_left_sidebar() {
     const rendered_sidebar = render_left_sidebar({
         is_guest: page_params.is_guest,
+        development_environment: page_params.development_environment,
```

Comment:
Let's put a comment here that we need this in the context because we want `inbox` to only be available in development for now, so we can remember why it's here for when we want to remove it.